client/src/pages/feedback/Feedback.scss

```diff
@@ -17,3 +17,17 @@
 .maxlength_warning {
   margin-top: 8px;
 }
+
+.modal {
+  &__header {
+    background-color: var(--section-bg-color);
+    color: var(--text-color);
+    &.modal-header {
```

Comment:
you have `.modal__header` className, why you use similar className

Reply:
.modal-header is a bootstrap class